day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,23 @@
+var frogs = ["One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten"];
+var reverse = frogs.reverse();
```

Comment:
Very cool using the `.reverse()`. I didn't think it would change the original array, but it does! Because `frogs` is actually reversed, I don't think you need to assign it to a `reverse` variable that is then never used.